js/excel_exporter.d.ts

```diff
@@ -45,6 +46,33 @@ export interface ExcelDataGridCell {
     value?: any;
 }
 
+/**
+* @docid ExcelPivotGridCell
+* @namespace DevExpress.excelExporter
+* @type object
+* @inherits dxPivotGridPivotGridCell
+*/
+export interface ExcelPivotGridCell extends dxPivotGridPivotGridCell {
```

Comment:
Do we really need dxPivotGridPivotGridCell object in a handler? Will we change its api?
For DataGrid we did not inherited a similar interface from dxDataGridDataGridCell

Reply:
It's ok

Reply:
Yes. For the PivotGrid we already have a ready-made description for the cell properties. In the case of DataGrid widget such description was skipped and we couldn't use the similar interface inherit.